Automattic/wp-calypso
# Jetpack Manage: Adding the ability to multi select filters in the sites dashboard

## Note

This PR is not intended to be implemented as Dataviews will eventually allow implementation of multi-selection of filters, but exists as a 'just in case' should we need something at some stage, and could build on this.

## Proposed Changes

* This PR adds the ability to multi select filters.


Not implemented:

* The current default behaviour when clicking a filter name that was selected immediately before, is to reset the filter (to 'Needs Attention'). This does not yet have an override.
* Another issue is that if you navigate away from the filter selection, the filters will not show as selected when re-opening the filter dropdown until another one is selected.

Initially linked to https://github.com/Automattic/jetpack-manage/issues/323

## Testing Instructions

* To test, apply this PR and then from the filter dropdown select multiple filters. Each should show as selected. If you click 'Needs Attention' the others will be unselected. If you click either reset button, the options will be reset.
* You will likely notice the existing issues as well (notes above).

## Pre-merge Checklist
